app/bundles/ApiBundle/Form/Type/ClientType.php

```diff
@@ -70,7 +90,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         if (!$options['data']->getId()) {
             $builder->add(
                 'api_mode',
-                'choice',
+                ChoiceType::class,
```

Comment:
All the choice types must have the `'choices_as_values' => true` option and the labels must be flipped with values (using array_flip or you can manually flip them here).
We are supposed to use labels as keys from Symfony 2.7 and later.
Please see https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit#heading=h.imucd6w2ou47
and
https://symfony.com/doc/2.7/reference/forms/types/choice.html#choices-as-values